stream-chat-android-ui-components-sample/src/main/res/layout/fragment_chat.xml

```diff
@@ -44,14 +44,30 @@
         app:streamUiPinMessageEnabled="true"
         />
 
-    <io.getstream.chat.android.ui.message.input.MessageInputView
-        android:id="@+id/messageInputView"
+    <FrameLayout
+        android:id="@+id/messageComposerContainer"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/messageListView"
-        />
+        >
```

Comment:
For now, I would like to keep both message composers in the sample app so that it's easy to switch between them and compare the behavior.